packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -720,6 +720,20 @@ addFieldsDict(Posts, {
     type: Date,
     viewableBy: ['guests'],
   },
+  
+  joinEventLink: {
+    type: String,
+    hidden: (props) => !props.eventForm,
+    viewableBy: ['guests'],
+    insertableBy: ['members'],
+    editableBy: ['members', 'sunshineRegiment', 'admins'],
```

Comment:
I think the second and third permissions here are redundant, unless I'm missing something. I tried the setup below, and I could still edit the link as an admin:
```suggestion
    editableBy: ['members'],
```